sysutils/restic/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/restic/restic 0.9.4 v
+categories          sysutils
+
+license             BSD
+maintainers         nomaintainer
+
+description         Fast, secure, efficient backup program.
+
+long_description    Restic is a program that does backups right. Its design goals are: Easy, Fast, Verifiable, Secure, Efficient and Free.
+
+distname            restic-${version}
```

Comment:
The `golang 1.0` PortGroup can set `distname` for you (inherited from `github 1.0`), try remove this line.